test/nodes/test_ranker.py

```diff
@@ -499,3 +503,264 @@ def test_cohere_ranker_batch_multiple_queries_multiple_doc_lists(docs, mock_cohe
     assert isinstance(results[0], list)
     assert results[0][0] == docs[4]
     assert results[1][0] == docs[4]
+
+
+recency_tests_inputs = [
+    # Score method works as expected
+    pytest.param(
```

Comment:
We haven't used `pytest.param` but it I don't see a reason against it here. @silvanocerza what's your opinion on it?

Reply:
I love this to be fair. `pytest.parametrize()` calculates its own id for each test, `pytest.param()` is the best way to have nicely named parametrized tests.

Great idea @elundaeva. :)